src/app/address_separator/address_separator_main.py

```diff
@@ -1,27 +1,33 @@
 import pandas as pd
 from app.address_separator.utils.extract.detail.check.detail_data_check import detail_check
 from app.address_separator.utils.modify.operation import post_process
-from app.address_separator.utils.os.output import create_output_data
 from app.address_separator.utils.preprocess.operation import pretreatment
 from app.address_separator.utils.reformat.output_data_shaping import format_for_output
 from app.address_separator.utils.split.splitByAddressField import split_by_address_field
 
 
-def address_separator_main(CSV_DATA: pd.Series) -> pd.DataFrame:
+def address_separator_main(ORIGINAL_ADDRESS: str) -> dict[str, str]:
+    """
+    Parameters
+    ----------
+    ORIGINAL_ADDRESS : str
+        未処理の住所
 
-    formatted_address_data_array: list[str] = pretreatment(CSV_DATA)
+    Returns:
+    -------
+    dict[str, str]
+        出力ファイルの列名とそれに対応する文字列
+    """
 
-    splitted_address_data: dict[str, list[str]] = split_by_address_field(formatted_address_data_array, CSV_DATA)
+    formatted_address_data_array: str = pretreatment(ORIGINAL_ADDRESS)
 
+    splitted_address_data: dict[str, str] = split_by_address_field(formatted_address_data_array, ORIGINAL_ADDRESS)
```

Comment:
```suggestion
    splitted_address: dict[str, str] = split_by_address_field(formatted_address_data_array, ORIGINAL_ADDRESS)
```
これは過去の私が悪いのですが、名著『リーダブルコード』にもあるように、意味をなさない〜dataのような、命名はやめるべきです。

余力があれば、こちらについても修正してもらえると